client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -334,6 +340,13 @@ function getRedirectUrlForConciergeNudge( {
 		// being offered and so sold, to be inline with HE availability.
 		// To dial back, uncomment the condition below and modify the test config.
 		if ( 'offer' === abtest( 'conciergeUpsellDial' ) ) {
+			if ( hasPersonalPlan( cart ) ) {
+				if ( shouldShowOneClickTreatment ) {
+					const upgradeItem = hasMonthlyCartItem( cart ) ? 'premium-monthly' : 'premium';
+					return `/checkout/${ siteSlug }/offer-plan-upgrade/${ upgradeItem }/${ pendingOrReceiptId }`;
```

Comment:
Please add a test for this in https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js!

Reply:
Test added!